inst/sql/sql_server/CohortEntryBreakdown.sql

```diff
@@ -10,8 +10,8 @@ IF OBJECT_ID('tempdb..@store_table', 'U') IS NOT NULL
 SELECT domain_table,
   domain_field,
   concept.concept_id,
-	concept_count,
-	subject_count
+	sum(concept_count) as concept_count,
```

Comment:
I think it is more appropriate to do a max for subject count

sum for concept_count
max for subject_count

Reply:
After talking to @pbr6cornell I changed this to max. Still not sure what the cause is but the issue appears to be rare and this is probably an adequate fix.